src/main/kotlin/com/ajaxproject/telegrambot/bot/repository/CurrencyExchangeRepository.kt

```diff
@@ -0,0 +1,15 @@
+package com.ajaxproject.telegrambot.bot.repository
+
+import com.ajaxproject.telegrambot.bot.models.currency.MongoCurrency
+import org.bson.types.ObjectId
+
+interface CurrencyExchangeRepository {
+
+    fun findByCode(code: Int): List<MongoCurrency?>
```

Comment:
```suggestion
    fun findByCode(code: Int): List<MongoCurrency>
```

Can we apply this change?

Reply:
+